algorithm/cgal/cgal.cc

```diff
@@ -3460,19 +3586,9 @@ class Geometry {
   void emitPolygonsWithHoles(int nth, emscripten::val emit_plane,
                              emscripten::val emit_polygon,
                              emscripten::val emit_point) {
-    Transformation absolute_to_local_transform = transform(nth).inverse();
-    Plane absolute_plane = plane(nth);
-    Plane local_plane =
-        unitPlane(absolute_plane.transform(absolute_to_local_transform));
-    std::vector<Polygon_with_holes_2> polygonsWithHoles;
-    gps(nth).polygons_with_holes(std::back_inserter(polygonsWithHoles));
-    for (auto& polygon : polygonsWithHoles) {
-      polygon = transformPolygonWithHoles(polygon, absolute_plane, local_plane,
-                                          absolute_to_local_transform);
-    }
-    emitPlane(local_plane, emit_plane);
-    ::emitPolygonsWithHoles(polygonsWithHoles, emit_polygon, emit_point);
-    plane(nth) = local_plane;
+    assert(is_local_frame());
```

Comment:
This is a neat thing to have. Just as a way to catch anything that might slip through in an unexpected frame.